genologics/entities.py

```diff
@@ -61,11 +61,11 @@ def control(self):
         #    logger.info(value[1]+"->"+value[0].id+"->"+key)
         logger.info("\nHistory :\n\n")
         logger.info("Input\tProcess\tProcess info")
-        for key, dict in self.history.items():
+        for key, dict in list(self.history.items()):
```

Comment:
Here is an example of where 2to3 acts a little naive, because it has to be OK to just loop through the items as a generator.

Do you think we should fix it or start with just having the 2to3 output as generated?

Reply:
I change.